src/Countries/Malaysia.php

```diff
@@ -0,0 +1,562 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use DateTime;
+use IntlDateFormatter;
+use Spatie\Holidays\Exceptions\InvalidRegion;
+use Spatie\Holidays\Exceptions\InvalidYear;
+
+class Malaysia extends Country
+{
+    protected string $timezone = 'Asia/Kuala_Lumpur';
+
+    /** @var array<int, string> $regions */
+    protected array $regions = [
+        'jhr',
+        'kdh',
+        'ktn',
+        'kul',
+        'lbn',
+        'mlk',
+        'nsn',
+        'phg',
+        'png',
+        'prk',
+        'pls',
+        'pjy',
+        'sbh',
+        'swk',
+        'sgr',
+        'trg',
+    ];
+
+    protected function __construct(
+        protected ?string $region = null,
+    ) {
+    }
+
+    public function countryCode(): string
+    {
+        return 'my';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge(
+            $this->fixHolidays($year),
+            $this->variableHolidays($year)
+        );
+    }
+
+    /**
+     * @param int $year
+     * @return array<string, string>
+     */
+    protected function fixHolidays(int $year): array
```

Comment:
Could you format this like other countries do? The fixed dates are not in a separate method, but just in the allHolidays() method